modyn/config/schema/pipeline/trigger/performance/performance.py

```diff
@@ -0,0 +1,95 @@
+from typing import Literal
+
+from pydantic import Field, field_validator, model_validator
+
+from modyn.config.schema.base_model import ModynBaseModel
+from modyn.config.schema.pipeline.evaluation.config import EvalDataConfig
+from modyn.config.schema.pipeline.trigger.performance.criterion import (
+    PerformanceTriggerCriterion,
+)
+from modyn.const.types import ForecastingMethod, TriggerEvaluationMode
+
+
+class PerformanceTriggerEvaluationConfig(ModynBaseModel):
+    # max_samples_per_evaluation: int = Field(
+    #     1000,
+    #     description=(
+    #         "The maximum number of samples that should be used for the evaluation. If more are available, "
+    #         "downsampling will be used"
+    #     ),
+    # )  # TODO: currently not yet supported by the evaluator, we use a evaluation dataset that is sufficiently small
```

Comment:
why is something to be supported by the evaluator? couldnt the trigger subsample the set?

Reply:
Iirc we only filter on the elevator side by specifying the first and last timestamp of the evaluation interval. As we currently don't use the evaluator at all, we can surely sample on the supervisor side. I also don't want to adjust the grpc interface again at this point for time reasons.